ror.yml

```diff
@@ -0,0 +1,147 @@
+# Recipe for Ruby on Rails collections
+---
+rh-ror50:
+  name: Ruby on Rails 5.0
+  requires: [rh-ruby23, rh-nodejs4, rh-mongodb32]
+  # install: Copr - Settings - Build options - Click "Edit" on target
+  #   Packages
+  install:
+    - scl-utils-build
+    - rh-ruby23-scldevel
+    - rh-ror50-build
+  packages:
+    - rh-ror50
+    # Packages required by RSpec, Cucumber
+    - rubygem-rspec
+    - rubygem-rspec-core:
+        replaced_macros:
+          need_bootstrap_set: 1
+    - rubygem-rspec-support:
+        replaced_macros:
+          need_bootstrap_set: 1
+    - rubygem-diff-lcs:
+        macros:
+          _with_bootstrap: 1
```

Comment:
> I would wish that the flag style was same in the packages.

That's great. I think we could make the change then.

> So, did you mean that you wanted me to change the flag style to your favorite one for both target Fedora packages and my rh-ror50 SCL packages this time?

I think it's up to you- if/when you have some "spare cycles" for it. I would prefer to have the change in Fedora first, but It's not something that affects the user, so this change should IMO come with some package patch/rebase. In SCL it helps you with the rebuild this yaml file is used for. Therefore it could just contain some kind of "mark" for bootstrapping, and no macro name at all.

> There is another bootstrap flag style right now at least in Perl package.

I am afraid I cannot see your point here, as this name is perl-specific. It is just a macro name after all. In this case it would especially help us to unify it in Rails 5 SCL, no matter how the macro is named. I would stick to Fedora guidelines in this case, with the exception of negating the value (I like @voxik's solution).

What do you propose, then?